tracer/test/Datadog.Trace.Tests/Configuration/ExporterSettingsTests.cs

```diff
@@ -142,8 +194,8 @@ private ExporterSettings Setup(Func<string, bool> fileExistsMock, params string[
 
             foreach (var item in config)
             {
-                var parts = item.Split(':');
-                configNameValues.Add(parts[0], parts[1]);
+                var separatorIndex = item.IndexOf(':');
```

Comment:
Since test data (urls!) can clearly have `:` chars, do you think we should switch this to use `=` instead? Or better yet, use separate strings so we don't need to split them as part of the test?

```csharp
private ExporterSettings Setup(Func<string, bool> fileExistsMock, params (string Key, string Value)[] config)
```

This could have led to subtle bugs in the tests. Something like `A:B:C` which was meant to be `A` and `B:C` would have been interpreted as `A` and `B` (and ignored `C`). A test could have passed by accident, for example, giving us false confidence.

Reply:
> Since test data (urls!) can clearly have : chars

Yes that's why I made this change, the previous implementation didn't work. 
But I considered it was enough as I don't see us putting a `:` in the key name in the tests. We're not that 😈 . 
But I'll look into using your proposal, as it's even simpler